server/specs/db.test.js

```diff
@@ -0,0 +1,119 @@
+const db = require('../db.js');
+
+
+describe('DB', () => {
+  
+  test('Should return an array when asked for similar listings', (done) => {
+    db.getSimilarListings(1, (err, result) => {
+      expect(Array.isArray(result)).toBe(true);
+      done();
```

Comment:
@jennqiao can I know how do you get the keywords for similar listings

Reply:
Yup! You can sort of see it in the next db test. I take the keywords from the listing with the same ID, and then find other listings that have those keywords listed in their own keywords array.